src/main/kotlin/ch/bfh/habits/entities/Habit.kt

```diff
@@ -10,6 +10,7 @@ import javax.persistence.*
 class Habit (
     var title: String,
     var description: String,
+    var frequency: Long? = null,
```

Comment:
This is not enough? We also need an enum with the values DAY, WEEK, MONTH:

See https://github.com/bfhmea4/mea4_01_habits/issues/148